### PR TITLE
SQuAD dataset and SQuAD model

### PR DESCRIPTION
**Pull request by yurakuratov**
This pull request adds SQuAD dataset support and SQuAD model, which achieves ~79 F-1 score.

### CODE REVIEW

File: deeppavlov/configs/squad/squad.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "name": "squad_dataset_reader",
+    "data_path": "squad/"
+  },
+  "dataset_iterator": {
+    "name": "squad_iterator",
+    "seed": 1337,
+    "shuffle": true
+  },
+  "chainer": {
+    "in": ["c_raw", "q_raw"],
```
Comment (yoptar) [inline deeppavlov/configs/squad/squad.json]: Maybe rename it to `context` and `question`? Interact would look better.

Comment (seliverstov) [reply deeppavlov/configs/squad/squad.json]: Agree. Meaningful names are much better.

File: deeppavlov/models/squad/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+To run model in interact mode run the following command:
+ ```bash
+python -m deeppavlov.deep interact deeppavlov/configs/squad/squad.json
+```
+Model will ask you to type in context and question.
+
+## Pretrained model on SQuAD
+Model is available at the following [[link]](lnsigo.mipt.ru/export/deeppavlov_data/squad_model.tar.gz)
+
+It achieves ~79 F-1 score and ~70 EM on dev set.
```
Comment (yoptar) [inline deeppavlov/models/squad/README.md]: Position on the leaderboard and comparison with other models would be nice.

File: deeppavlov/models/squad/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+## Pretrained model on SQuAD
+Model is available at the following [[link]](lnsigo.mipt.ru/export/deeppavlov_data/squad_model.tar.gz)
+
+It achieves ~79 F-1 score and ~70 EM on dev set. 
+
+## Training on SDSJ Task B
+If you want to train this model on SDSJ Task B then you should follow these steps:
+* Convert data to SQuAD format
+* Use Russian [word](http://lnsigo.mipt.ru/export/embeddings/ft_native_300_ru_wiki_lenta_nltk_word_tokenize/ft_native_300_ru_wiki_lenta_nltk_word_tokenize.vec) and [character](http://lnsigo.mipt.ru/export/embeddings/ft_native_300_ru_wiki_lenta_nltk_word_tokenize-char.vec) embeddings
```
Comment (yoptar) [inline deeppavlov/models/squad/README.md]: Can you also add what parameters in config one needs to change to make it work?

File: main.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+for x, y in els[:3]:
+    print(x)
+    print(y)
+    out = chainer(x, y)
+    #print(out.shape)
+    for el in out:
+        print(el)
+
+x, y = dataset.iter_all()
```
Comment (yoptar) [inline main.py]: This file should be removed

File: deeppavlov/configs/squad/squad.json (hunk 1 above)
Comment (yurakuratov) [reply deeppavlov/configs/squad/squad.json]: renamed

File: deeppavlov/models/squad/README.md (hunk 2 above)
Comment (yurakuratov) [reply deeppavlov/models/squad/README.md]: added link to SQuAD leaderboard

File: deeppavlov/models/squad/README.md (hunk 3 above)
Comment (yurakuratov) [reply deeppavlov/models/squad/README.md]: fixed

File: main.py (hunk 4 above)
Comment (yurakuratov) [reply main.py]: done

**Pull request merged**